eks-hybrid-iam-ra-setup.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+set -eux -o pipefail
```

Comment:
redirect stdout to stderr, since a lot of the echoes are going to `stderr`, and won't be visible otherwise
```suggestion
exec >&2
set -eu -o pipefail
```